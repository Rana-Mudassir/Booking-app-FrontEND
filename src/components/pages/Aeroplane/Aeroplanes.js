import React from 'react';
import AeroplaneList from './AeroplaneList'; // The component created above

const MainComponent = () => (
  <div>
    <AeroplaneList />
  </div>
);

export default MainComponent;
