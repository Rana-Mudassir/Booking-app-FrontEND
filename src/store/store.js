import { configureStore } from '@reduxjs/toolkit';
import loginReducer from './authentication/login';
import signupReducer from './authentication/signup';
import aeroplaneReucer from './aeroplane/aeroplane';
import aeroplaneslistReducer from './aeroplaneList/aeroplaneListSlice';
import selectedAirplaneSliceReducer from './aeroplaneList/selectedAirplaneSlice';

const store = configureStore({
  reducer: {

    login: loginReducer,
    signup: signupReducer,
    aeroplane: aeroplaneReucer,
    aeroplaneslist: aeroplaneslistReducer,
    selectedAirplane: selectedAirplaneSliceReducer,
  },
});

export default store;
