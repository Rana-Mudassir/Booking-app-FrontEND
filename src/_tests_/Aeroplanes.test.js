import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../store/store';
import Aeroplanes from '../components/pages/Aeroplane/Aeroplanes';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <Aeroplanes />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
