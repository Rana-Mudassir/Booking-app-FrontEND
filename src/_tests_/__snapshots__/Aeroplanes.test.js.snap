// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="AeroList"
  >
    <h2>
      Aeroplane List
    </h2>
    <ul
      className="list"
    />
  </div>
</div>
`;
